java/src/jmri/jmrit/beantable/oblock/OBlockTableModel.java

```diff
@@ -616,27 +619,6 @@ public void setValueAt(Object value, int row, int col) {
                 block.setPermissiveWorking((Boolean) value); // compare to REPORT_CURRENTCOL
                 fireTableRowsUpdated(row, row);
                 return;
-            case WARRANTCOL:
-                Warrant warrant = block .getWarrant();
-                jmri.jmrit.logix.WarrantManager mgr = InstanceManager
-                            .getDefault(jmri.jmrit.logix.WarrantManager.class);
-                Warrant newWarrant = mgr.getWarrant((String)value);
-                if (warrant != null && !warrant.equals(newWarrant)) {
-                    block.deAllocate(warrant);
-                    if (newWarrant != null) {
-                        String msg = block.allocate(newWarrant);
-                        if (msg != null) {
-                            JOptionPane.showMessageDialog(null, msg,
-                                    Bundle.getMessage("ErrorTitle"), JOptionPane.WARNING_MESSAGE);                    
-                        }                    
-                    }
-                }
-                fireTableRowsUpdated(row, row);
-                return;
-            case VALUE:
-                block.setValue(value);
-                fireTableRowsUpdated(row, row);
-                return;
```

Comment:
@petecressman 
I haven't used warrant so I'm asking to understand the code.
If I understand this correctly, the user was able to edit warrant in the OBlock table but isn't able to do it any more?

Reply:
These are non-editable columns. They display the warrant name that has allocated the OBlock for its use and the train name of a train that is occupying an OBlock, if known. The information comes from conditions on the layout and should not overridden manually. This mistaken code that should be removed.